website/docusaurus.config.js

```diff
@@ -347,41 +347,47 @@ const config = {
             title: 'Documentation',
             items: [
               {
-                label: 'Installing Podman Desktop',
+                label: 'Installing',
                 to: '/docs/installation',
               },
               {
                 label: 'Onboarding for containers',
                 to: '/docs/onboarding/containers',
               },
-              {
-                label: 'Onboarding for Kubernetes',
-                to: '/docs/onboarding-for-kubernetes',
-              },
               {
                 label: 'Working with containers',
                 to: '/docs/working-with-containers',
               },
               {
-                label: 'Migrating from Docker',
-                to: '/docs/migrating-from-docker',
+                label: 'Working with Kubernetes',
+                to: '/docs/kubernetes',
               },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
               {
-                label: 'Working with Compose',
-                to: '/docs/compose',
+                label: 'Discord',
+                href: 'https://discord.com/channels/852634929845239818/1085121643782873118',
               },
+
               {
-                label: 'Working with Kubernetes',
-                to: '/docs/kubernetes',
+                label: 'IRC',
+                href: 'https://web.libera.chat/#podman-desktop',
+              },
+              {
+                label: 'Matrix',
+                href: 'https://matrix.to/#/#podman-desktop:fedoraproject.org',
               },
               {
-                label: 'Troubleshooting',
-                to: '/docs/troubleshooting',
+                label: 'Kubernetes Slack',
```

Comment:
When mentioning, we need to make sure that Kubernetes-discussions are down on the Kubernetes slack. It was mentioned how podman desktop "general" chats should be on Discord / IRC / Matrix, while "kubernetes" chats on the k8s slack. This is because k8s slack is a HUGE community and is beneficial for showcasing PD.